src/theme.ts

```diff
@@ -0,0 +1,67 @@
+import { Dimensions } from "react-native";
+import { Colors, Fonts, Sizes, Weights } from "./global";
+
+
+const { width, height } = Dimensions.get("window");
+
+export const COLORS: Colors = {
+  // default font color
+  font: "#000000",
+
+  // base colors
+  primary: "#4630EB",
+  secondary: "#A3A1F7",
+  tertiary: "#FFE358",
+
+  // non-colors
+  black: "#000020",
+  white: "#FFFFFF",
+
+  // color variations
+  gray: "#535453",
+  error: "#DC3545",
+  warning: "#FFE358",
+  success: "#4CD964",
+  info: "#4DA1FF"
+};
+
+export const SIZES: Sizes = {
```

Comment:
The SIZES constant. It should be radius?

Reply:
SIZES is an object with values for different usage.
For example: **radius** is used for **borderRadius**

Reply:
Yes! I should have been clear than that. What I mean is, the SIZES.radius is the correct value for the failing test below?